File: support_tools/swigwin-3.0.12/Examples/test-suite/python/li_std_stream_runme.py
from li_std_stream import *


a = A()

o = ostringstream()

o << a << " " << 2345 << " " << 1.435


if o.str() != "A class 2345 1.435":
    print "\"%s\"" % (o.str(),)
    raise RuntimeError

File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/li_std_stream_runme.rb
#!/usr/bin/env ruby
#
# Simple test of std::ostringstream.
#
# 
# 
# 
#

require 'swig_assert'

require 'li_std_stream'
include Li_std_stream

swig_assert_each_line(<<'EOF', binding)

a = A.new
o = Ostringstream.new
o << a << " " << 2345 << " " << 1.435
o.str == "A class 2345 1.435"

EOF
